src/us/kbase/workspace/kbase/admin/AdministrationCommandSetBuilder.java

```diff
@@ -0,0 +1,486 @@
+package us.kbase.workspace.kbase.admin;
```

Comment:
The essence of the PR is moving the individual admin commands from WorkspaceAdministration to this builder so eventually the builder can install alternate sets of commands depending the needs of the service.